popmon/visualization/histogram_section.py

```diff
@@ -193,7 +193,7 @@ def _plot_histograms(feature, date, hc_list, hist_names, top_n, max_nbins=1000):
 
     # make plot. note: slow!
     if hc_list[0].n_dim == 1:
-        if all(h.size == 0 for h in hc_list):
```

Comment:
Did you use histogrammar v1.0.28? It needs that. 
(Not sure this is right otherwise.)